lazycoder1412/EvaluateDivision.cpp

```diff
@@ -0,0 +1,34 @@
+class Solution {
+public:
+    map<string,vector<pair<string,double> > >graph;
+    map<string,bool> vis;
+    vector<double> ans;
+    double dfs(string start,string end){
+        vis[start]=true;
+        if(start==end)
+            return 1.0;
+        for(auto nodes: graph[start])
```

Comment:
`const auto &nodes` is more efficient since you don't create copies of the nodes.